governance/votes/Executive vote - July 25, 2024.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 1 Setup - July 25, 2024
+summary: Setting up the Lightweight Peg Stability Module (LITE-PSM-USDC-A) for Phase 1.
+date: 2025-07-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] LITE-PSM-USDC-A Phase 1 Setup - July 25, 2024
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+
+- LITE-PSM-USDC-A Phase 1 actions:
+  - PSM-USDC-A Debt Ceiling Instant Access Module (DC-IAM) parameters will be updated.
```

Comment:
Add link for [DC-IAM](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-debt-ceiling-instant-access-module-dc-iam-)